libr/core/cmd_anal.c

```diff
@@ -2962,6 +2974,14 @@ static int cmd_anal_fcn(RCore *core, const char *input) {
 			eprintf ("Usage: afo[?sj] ([name|offset])\n");
 			break;
 		case 'j':
+			{
+				ut64 addr = core->offset;
+				RAnalFunction *fcn = r_anal_get_fcn_in (core->anal, addr, R_ANAL_FCN_TYPE_NULL);
+				if (fcn) {
+					r_cons_printf ("{\"address\":\"0x%08" PFMT64x "\"}\n", fcn->addr);
```

Comment:
there can be more than one function in the given offset, i have opened a pr to work on that, so this may change to iterate a list, also will be good to use the pj api instead of a raw println, addresses are usually printed in decimal, not as a string.

Reply:
Ok I implemented it now using pj API (can remove again if we rather want to wait until your PR is done, but the feature would be nice to have in my script anyways)